backend/pyproject.toml

```diff
@@ -25,14 +25,13 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
 ]
 dependencies = [
-    "Products.CMFPlone==6.1.1",
+    "kitconcept.core==1.0.0a1",
     "collective.person",
     "plone.api",
     "plone.distribution",
-    "plone.restapi>=9.13.2",
-    "plone.volto>=5.0.2",
+    "plone.restapi",
```

Comment:
We need the new release I made today (9.14.0) in order to support adapters that add info to the `@site` endpoint.

Reply:
I will update that on kitconcept.core